src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/_client_factory.py

```diff
@@ -0,0 +1,14 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+def cognitiveservices_client_factory(_):
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
+    from azure.mgmt.cognitiveservices import CognitiveServicesManagementClient
+    return get_mgmt_service_client(CognitiveServicesManagementClient, location='notused').cognitive_services_accounts
+
+def cognitiveservices_account_client_factory(_):
```

Comment:
Refactor by adding  `get_ cognitiveservices_management_client` and then use them in `cognitiveservices_account_client_factory` & other operations

Reference: https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_client_factory.py#L8